lib/msal-angular/src/msal.interceptor.spec.ts

```diff
@@ -105,41 +104,43 @@ describe('MsalInterceptor', () => {
     initializeMsal();
   });
 
-  it("throws error if incorrect interaction type set in interceptor configuration", (done) => {
+  it("throws error if incorrect interaction type set in interceptor configuration", fakeAsync(() => {
     testInteractionType = InteractionType.Silent;
     initializeMsal();
 
+    let response;
+
     httpClient.get("https://graph.microsoft.com/v1.0/me").subscribe({
       error: (error) => {
-        expect(error.errorCode).toBe("invalid_interaction_type");
-        expect(error.errorMessage).toBe("Invalid interaction type provided to MSAL Interceptor. InteractionType.Popup, InteractionType.Redirect must be provided in the msalInterceptorConfiguration");
-        testInteractionType = InteractionType.Popup;
-        done();
+        response = error;
       }
     });
-  });
 
-  it("does not attach authorization header for unprotected resource", (done) => {
+    tick();
```

Comment:
using fakeAsync() might lead to some complications with code refactoring down the line, using async() in place of done()  might be more consistent with rest of the test base.

Reply:
Yes, in this case I was a bit in doubt. It is fine, I guess, to use async as this doesn't require these artificial setTimeouts to block and this runs just as fast with a done().